File: Week1Exe/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    // Prompt user for height of steps
    do
    {
        n = get_int("Heights: ");
    }
    while (n > 8 || n < 1);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j <= 2*n + 1; j++)
        {
            // Print spaces between steps
            if (j == n || j == n + 1)
            {
                printf(" ");
            }

            // Print Spaces on left side of left steps
            else if (i + j < n - 1)
            {
                printf(" ");
            }

            // Print right side steps
            else if (i - j > 1/2*n - n - 3)
            {
                printf("#");
            }
        }
        printf("\n");
    }



}

File: Week6/sentimental-mario-less/mario.py
from cs50 import get_int

# Prompt user for input
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Loop function
for i in range(height):
    for j in range(height):
        if i + j > height - 2:
            print("#", end="")
        else:
            print(" ", end="")
    print("")